luvit-loader.lua

```diff
@@ -197,7 +197,7 @@ local function loader(dir, path, bundleOnly)
         return package.loaded[key]
       end
       local code = bundle.readfile(fullPath)
-      local module = loadstring(code, key)()
+      local module = loadstring(code, key)(key)
```

Comment:
So even though this is a bundle path, we can sometimes get a real system path (like when luvi is being run against a directory on the disk).  See how this is done in luvit's require. https://github.com/luvit/luvit/blob/1e2b4593c5552a642f8bf31142f440848d8b226d/deps/require.lua#L292-L297